op-node/rollup/driver/conf_depth.go

```diff
@@ -33,7 +32,10 @@ func (c *confDepth) L1BlockRefByNumber(ctx context.Context, num uint64) (eth.L1B
 
 	// Don't apply the conf depth is l1Head is empty (as it is during the startup case before the l1State is initialized).
 	l1Head := c.l1Head()
-	if num == 0 || c.depth == 0 || num+c.depth <= l1Head.Number || l1Head.Hash == (common.Hash{}) {
+	if l1Head == (eth.L1BlockRef{}) {
```

Comment:
Are these 2 cases just split up for readability?

Reply:
yea